libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Extensions/PushNotificationManager.swift

```diff
@@ -89,8 +89,8 @@ extension PushNotificationManager {
 
 extension PushNotificationManager {
     @objc
-    func fetchAndStoreNotificationTypes(restClient: RestClient = RestClient.shared,
-                                 account: UserAccount? = UserAccountManager.shared.currentUserAccount) async throws {
+    public func fetchAndStoreNotificationTypes(restClient: RestClient = RestClient.shared,
```

Comment:
On the Template App, if this wasnt public, during compile time Xcode complaied it wasn't able to find the method in SFPushNotificationManager.